newsfragments/1526.feature.rst

```diff
@@ -0,0 +1,2 @@
+Some users are confused with the usage of `open_signal_receiver <https://trio.readthedocs.io/en/stable/reference-io.html#trio.open_signal_receiver>`__ being able to operate without any signal.
+This issue being fixed.
```

Comment:
The wording you have here is not clear about what changed, which is the main point of a newsfragment. I would say something like: Calling `open_signal_receiver` with no arguments used to succeed without listening for any signals. This was confusing, so now it raises `ValueError` instead.

Also this isn't really a "feature" - I would probably put this in the "bug fixes" section of the release notes (so name the newsfragment "1526.bugfix.rst").